### PR TITLE
created the hero component for the conference page

### PR DESCRIPTION
**Pull request by zorro1rr**
Also created bare sponsor page I could link to and edited the button/a-link styles in our global styles.

### CODE REVIEW

File: gatsby/src/components/ConferenceHero.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  }
+}
+`;
+
+function ConferenceHero() {
+  return (
+    <Hero>
+      <div id='hero-box'>
+        <div id='img-div'>
+          <img src={heroJS} alt='Utah JS Logo' id='hero-box-img' />
```
Comment (mikelnorth) [inline gatsby/src/components/ConferenceHero.js:86]: Can we use gatsby-image

You will make a page query to graphql using imageSharp

File: gatsby/src/components/Footer.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```
Comment (mikelnorth) [inline gatsby/src/components/Footer.js:1]: Remove this line

File: gatsby/src/pages/conference.js [hunk 3]
```diff
@@ -1,11 +1,16 @@
+/* eslint-disable prettier/prettier */
```
Comment (mikelnorth) [inline gatsby/src/pages/conference.js:1]: remove this line

File: gatsby/src/pages/conference.js [hunk 4]
```diff
@@ -1,11 +1,16 @@
...
 import React from 'react';
 
 import Layout from '../components/layout';
 import SEO from '../components/seo';
+import ConferenceHero from '../components/ConferenceHero';
+
+// import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (mikelnorth) [inline gatsby/src/pages/conference.js:8]: Delete this comment

File: gatsby/src/pages/sponsor.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable prettier/prettier */
```
Comment (mikelnorth) [inline gatsby/src/pages/sponsor.js:1]: same as above

File: gatsby/src/styles/main.js [hunk 6]
```diff
@@ -32,6 +32,30 @@ h5 {
...
+    background-color: #ff9000
+  }
+  .btn.btn-secondary {
+    color: #222;
+    background-color: #fae034;
+  }
+  .btn.btn-secondary:hover {
+    background-color: #fff28b
+  }
+  a{
```
Comment (mikelnorth) [inline gatsby/src/styles/main.js:50]: Moving this a tag styles into the styled component itself

**Review by mikelnorth — CHANGES_REQUESTED** [review]

File: gatsby/src/components/ConferenceHero.js (hunk 1 above)
Comment (mikelnorth) [reply gatsby/src/components/ConferenceHero.js:86]: You can do a shearch in the project for where we use imageSharp and gatsby-image to have some examples

**Comment by zorro1rr** [comment]
close to submit new changes

**Pull request closed without merging**